src/components/ModalWithForm.js

```diff
@@ -8,8 +8,14 @@ export default class ModalWithForm extends Modal {
     this._handleFormSubmit = handleFormSubmit;
   }
 
-  close() {
-    super.close(this._modal);
+  setLoading(isLoading, text) {
+    const modalButton = this._modal.querySelector(".modal__button");
+    console.log(modalButton);
+    modalButton.textContent = isLoading ? text : "Save";
+  }
+
+  reset() {
+    this._inputList.textContent = "";
```

Comment:
`    this._inputList.textContent = "";
`

You can easily clear the inputs with `reset`. Every **form** has this method. 

```js

    this._modalForm.reset()

```

Reply:
Ahhh this makes much more sense! I was previously trying to call reset() inside index.js or in other spots but it logged not a function. Reset being a prototype of all forms is what I was messing thank you!